### ISSUE TITLE
Bootstrap failed: 125: Domain does not support specified action

### ISSUE THREAD

**Issue by 101Dude** [Jackett/Jackett#14822]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
2023-11-05 23:21:09.7881 Info Environment version: 6.0.22 (/usr/local/Jackett/)
2023-11-05 23:21:09.7936 Info OS version: Unix 14.1.0 (64bit OS) (64bit process)
2023-11-05 23:21:09.7995 Info Jackett variant: CoreMacOs
2023-11-05 23:21:09.7995 Info Running in Docker: No
2023-11-05 23:21:09.8003 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
2023-11-05 23:21:09.8003 Info App config/log directory: /Users/john/.config/Jackett
2023-11-05 23:21:09.8011 Info Using proxy: Disabled
2023-11-05 23:21:09.8011 Info Using FlareSolverr: http://127.0.0.1:8191
2023-11-05 23:21:09.8022 Info Using HTTP Client: HttpWebClient2
```

### Description

jackett wasn't updating. Recommendation in logs was to reinstall the application. No matter which directory I unzipped jackett into and then clicked on `install_service_macos` I received the error listed below.

Oddly the service is listed as running, but it is not.

I wanted to post this... based off the upgrade bug alone. Sort of been a off and on again issue for me.

I tried the following local directories for unzipping the jackett directory
```
/usr/bin/jackett
/usr/local/jackett
/Applications
/Users/john/Applications/Jackett
```

I don't have SIP running or Gatekeeper. Restarting the machine does't solve.

I can run jackett from Terminal!

```
01:06:42.146 at iMac191 ~ /Users/john/Applications/Jackett/jackett 
11-06 01:07:47 Info Starting Jackett v0.21.1138
11-06 01:07:47 Info Environment variable MONO_GAC_PREFIX is present
11-06 01:07:48 Info Environment version: 6.0.24 (/Users/john/Applications/Jackett/)
11-06 01:07:48 Info OS version: Unix 14.1.0 (64bit OS) (64bit process)
11-06 01:07:48 Info Jackett variant: CoreMacOs
11-06 01:07:48 Info Running in Docker: No
11-06 01:07:48 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
11-06 01:07:48 Info App config/log directory: /Users/john/.config/Jackett
11-06 01:07:48 Info Using proxy: Disabled
11-06 01:07:48 Info Using FlareSolverr: http://127.0.0.1:8191
11-06 01:07:48 Info Using HTTP Client: HttpWebClient2
11-06 01:07:48 Info Loading Native indexers ...
11-06 01:07:48 Info Loaded 73 Native indexers.
11-06 01:07:48 Info Loading Cardigann indexers from: /Users/john/.config/cardigann/definitions/, /etc/xdg/cardigan/definitions/, /Users/john/Applications/Jackett/Definitions
11-06 01:07:49 Info Loaded 524 Cardigann indexers.
11-06 01:07:49 Info Loaded 597 indexers in total
11-06 01:07:49 Info Adding aggregate indexer ('all' indexer) ...
11-06 01:07:49 Info Adding filter indexer ('type:public' indexer) ...
11-06 01:07:49 Info Adding filter indexer ('type:private' indexer) ...
11-06 01:07:49 Info Adding filter indexer ('type:semi-public' indexer) ...
11-06 01:07:49 Info Jackett startup finished in 1.941 s
Hosting environment: Production
Content root path: /Users/john/Applications/Jackett/Content
Now listening on: http://127.0.0.1:9117
Application started. Press Ctrl+C to shut down.
```

These are permissions for items in LaunchAgents directory. Jackett's default permissions for `org.user.Jackett.plist` may be the issue here? In comparison to the other agents.

```
01:12:40.114 at iMac191 ~/Library/LaunchAgents ls -aleo@
total 64
drwxr-xr-x    9 john   288 Nov  6 01:10 .
drwx------+ 118 john  3776 Nov  2 01:25 ..
 0: group:everyone deny delete
-rw-r--r--@   1 john  6148 Nov  6 00:35 .DS_Store
	com.apple.FinderInfo	  32 
-rwxr-xr-x    1 john   684 Sep 13 19:28 com.dropbox.DropboxMacUpdate.agent.plist
-rw-r--r--@   1 john   808 Sep  8 12:35 com.google.keystone.agent.plist
	com.apple.TextEncoding	  15 
-rw-r--r--@   1 john   914 Sep  8 12:35 com.google.keystone.xpcservice.plist
	com.apple.TextEncoding	  15 
-rw-r--r--@   1 john   540 Oct 10 23:39 com.privateinternetaccess.vpn.client.plist
	com.apple.TextEncoding	  15 
	com.apple.provenance	  11 
-rwxrwxrwx@   1 john   670 Nov  6 01:12 org.user.Jackett.plist
	com.apple.TextEncoding	  15 
-rw-r--r--@   1 john   670 Oct  6 15:15 org.virtualbox.vboxwebsrv.plist
	com.apple.TextEncoding	  15 
	com.apple.cs.CodeDirectory	 149 
	com.apple.cs.CodeRequirements	 188 
	com.apple.cs.CodeRequirements-1	 185 
	com.apple.cs.CodeSignature	9090 
	com.apple.provenance	  11 
```

### Logged Error Messages

```text
23:50:25.158 at iMac191 ~ su root
Password:
sh-3.2# /Users/john/Applications/Jackett/install_service_macos 
Removing Jackett executable and all .dylib and .dll files from quarantine...
Launching agent...
Bootstrap failed: 125: Domain does not support specified action
ERROR: Agent could not be loaded. Please open an issue on https://github.com/Jackett/Jackett/issues and paste the output.
*Jackett directory*: /Users/john/Applications/Jackett
*LaunchAgents permissions*:
drwxrwxrwx   3 root  wheel   96 Nov  5 23:50 .
drwx------  22 root  wheel  704 Nov  5 23:50 ..
-rw-rw-rw-   1 root  wheel  706 Nov  5 23:50 org.user.Jackett.plist
sh-3.2# 

2023-11-05 23:24:20.6138 Info Jackett startup finished in 2.001 s
2023-11-05 23:25:06.3741 Info Checking for updates... Jackett variant: CoreMacOs
2023-11-05 23:25:07.0430 Info New release found. Current version: v0.21.976 New version: v0.21.1138
2023-11-05 23:25:07.0434 Info Downloading release v0.21.1138 It could take a while...
2023-11-05 23:25:20.8969 Info Starting updater: /var/folders/83/phz93cq950n_370bvsgcttjc0000gn/T/JackettUpdate-v0.21.1138-638348235195964990/Jackett/JackettUpdater --Path "/usr/local/Jackett" --Type "Console" --Args " " --KillPids "42593"
2023-11-05 23:25:20.9161 Info Updater started process id: 44648
2023-11-05 23:25:20.9164 Info Exiting Jackett..
2023-11-05 23:25:20.9267 Info Jackett stopped
2023-11-05 23:25:21.1567 Info Starting Jackett v0.21.976
2023-11-05 23:25:21.6563 Info Environment version: 6.0.22 (/usr/local/Jackett/)
2023-11-05 23:25:21.6624 Info OS version: Unix 14.1.0 (64bit OS) (64bit process)
2023-11-05 23:25:21.6632 Info Jackett variant: CoreMacOs
2023-11-05 23:25:21.6632 Info Running in Docker: No
2023-11-05 23:25:21.6642 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
2023-11-05 23:25:21.6642 Info App config/log directory: /Users/john/.config/Jackett
2023-11-05 23:25:21.6642 Info Using proxy: Disabled
2023-11-05 23:25:21.6653 Info Using FlareSolverr: http://127.0.0.1:8191
2023-11-05 23:25:21.6662 Info Using HTTP Client: HttpWebClient2
2023-11-05 23:25:21.6696 Info Loading Native indexers ...
2023-11-05 23:25:21.7067 Info Loaded 74 Native indexers.
2023-11-05 23:25:21.7165 Info Loading Cardigann indexers from: /Users/john/.config/cardigann/definitions/, /etc/xdg/cardigan/definitions/, /usr/local/Jackett/Definitions
2023-11-05 23:25:22.7836 Info Loaded 522 Cardigann indexers.
2023-11-05 23:25:22.7836 Info Loaded 596 indexers in total
2023-11-05 23:25:22.7861 Info Adding aggregate indexer ('all' indexer) ...
2023-11-05 23:25:22.7912 Info Adding filter indexer ('type:public' indexer) ...
2023-11-05 23:25:22.7972 Info Adding filter indexer ('type:private' indexer) ...
2023-11-05 23:25:22.8073 Info Adding filter indexer ('type:semi-public' indexer) ...
2023-11-05 23:25:22.8129 Info Adding filter indexer ('tag:mvgroup' indexer) ...
2023-11-05 23:25:22.8132 Info Adding filter indexer ('tag:torrentleech' indexer) ...
2023-11-05 23:25:22.8171 Info Deleting JackettUpdate temp files from /var/folders/83/phz93cq950n_370bvsgcttjc0000gn/T/JackettUpdate-v0.21.1138-638348235195964990
2023-11-05 23:25:22.8916 Error An error occurred during the last update. If this error occurs again, you need to reinstall Jackett following the documentation. If the problem continues after reinstalling, report the issue and attach the Jackett and Updater logs.
2023-11-05 23:25:22.9790 Info Jackett startup finished in 2.027 s
2023-11-05 23:27:22.8108 Info Jackett stopped
2023-11-05 23:49:35.4900 Info Starting Jackett v0.21.1138
2023-11-05 23:49:36.0138 Info Environment version: 6.0.24 (/usr/local/Jackett/)
2023-11-05 23:49:36.0193 Info OS version: Unix 14.1.0 (64bit OS) (64bit process)
2023-11-05 23:49:36.0258 Info Jackett variant: CoreMacOs
2023-11-05 23:49:36.0258 Info Running in Docker: No
2023-11-05 23:49:36.0274 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
2023-11-05 23:49:36.0279 Info App config/log directory: /Users/john/.config/Jackett
2023-11-05 23:49:36.0290 Info Using proxy: Disabled
2023-11-05 23:49:36.0290 Info Using FlareSolverr: http://127.0.0.1:8191
2023-11-05 23:49:36.0305 Info Using HTTP Client: HttpWebClient2
2023-11-05 23:49:36.0345 Info Loading Native indexers ...
2023-11-05 23:49:36.0682 Info Loaded 73 Native indexers.
2023-11-05 23:49:36.0706 Info Loading Cardigann indexers from: /Users/john/.config/cardigann/definitions/, /etc/xdg/cardigan/definitions/, /usr/local/Jackett/Definitions
2023-11-05 23:49:37.1066 Info Loaded 524 Cardigann indexers.
2023-11-05 23:49:37.1068 Info Loaded 597 indexers in total
2023-11-05 23:49:37.1138 Info Adding aggregate indexer ('all' indexer) ...
2023-11-05 23:49:37.1210 Info Adding filter indexer ('type:public' indexer) ...
2023-11-05 23:49:37.1210 Info Adding filter indexer ('type:private' indexer) ...
2023-11-05 23:49:37.1257 Info Adding filter indexer ('type:semi-public' indexer) ...
2023-11-05 23:49:37.1257 Info Adding filter indexer ('tag:mvgroup' indexer) ...
2023-11-05 23:49:37.1267 Info Adding filter indexer ('tag:torrentleech' indexer) ...
2023-11-05 23:49:37.2156 Info Jackett startup finished in 1.980 s
2023-11-05 23:49:37.8602 Info Jackett stopped
```

**Comment by garfield69** [comment]
@majorsl @thebluepotato  We've got another mac osx mystery and could use your experience. Could you take a look? Thanks.

**Comment by 101Dude** [comment]
The default permissions on the launch agent plist are incorrect.

Changing them to:
```
-rwxr-xr-x@   1 john   670 Nov  6 01:12 org.user.Jackett.plist
	com.apple.TextEncoding	  15 
```
Immediately the process started and the jackett binary was added to "background items". I then went to System Preferences and toggled Jackett on and off and the process started.

![Screenshot 2023-11-06 at 1 19 30 AM](https://github.com/Jackett/Jackett/assets/142523670/62f50913-3906-4ad7-92ba-e526581af7df)
